### DISCUSSION TITLE
Is there a way for this to work with Unity?

### DISCUSSION THREAD

**Discussion by copypasteearth** [vvvv/SVG#976] [Q&A]
Im wondering if I can build this for unity, I tried to build the dll and its telling me 

Assembly 'Assets/Plugins/Svg.dll' will not be loaded due to errors:
Unable to resolve reference 'System.Drawing.Common'. Is the assembly missing or incompatible with the current platform?
Reference validation can be disabled in the Plugin Inspector.
Unable to resolve reference 'System.Memory'. Is the assembly missing or incompatible with the current platform?
Reference validation can be disabled in the Plugin Inspector.
Unable to resolve reference 'Fizzler'. Is the assembly missing or incompatible with the current platform?
Reference validation can be disabled in the Plugin Inspector.

Thank you for your time

**Comment by H1Gdev** [comment]
Are there any related DLLs ?
Svg.dll references some DLLs.

**Comment by copypasteearth** [comment]
I think unity cannot use System.Drawing.Common

Unity uses its own drawing library

**Comment by tkideneB** [comment]
I am unsure if that answers the question, but as an idea, you can import SVGs at runtime (https://answers.unity.com/questions/1701431/read-svg-from-disk-using-the-new-vector-graphics-p.html).

And if you want to edit them you could use a very simple editing library like SvgLib (https://github.com/huysentruitw/svglib other package hope it is ok ?).
